Python/Product/VSCommon/Logging/PythonToolsLoggerData.cs

```diff
@@ -167,6 +167,18 @@ static class PackageInstallInfoBarActions {
         public const string Ignore = "Ignore";
     }
 
+    sealed class PythonVersionNotSupportedInfoBarInfo : PythonToolsLoggerData {
+        public string Action { get; set; }
+        public string Context { get; set; }
+        public Version PythonVersion { get; set; }
```

Comment:
this needs to be a string, only simple types allowed, see similar cases in this file